tools/ALARAJOYWrapper/reaction_data.py

```diff
@@ -81,9 +81,8 @@ def emission_breakdown(emitted_particles):
     }
 
     # Handle gas production totals (MTs 203-207) of the format Xz
-    gases = ['p', 'd', 't', 'h', 'a']
     if 'X' in emitted_particles:
-        for gas in gases:
+        for gas in list(NP_dict.keys())[2:]:
```

Comment:
Readability: probably useful to still define
```
gases = list(NP_dict.keys())[2:]
```
maybe immediately following the definition of `NP_dict`